trade-finance-manager-api/src/v1/emails/AIN-MIN-confirmation/gef-facilities-list.test.js

```diff
@@ -82,20 +85,60 @@ describe('generate AIN/MIN confirmation email facilities list email variable/str
 
       expect(result).toEqual(expected);
     });
+
+    it('should format isUsingFacilityEndDate to Yes when true', () => {
+      const result = facilityFieldsObj({ isUsingFacilityEndDate: true });
+
+      expect(result).toEqual({ isUsingFacilityEndDate: 'Yes' });
+    });
+
+    it('should format isUsingFacilityEndDate to No when false', () => {
+      const result = facilityFieldsObj({ isUsingFacilityEndDate: false });
+
+      expect(result).toEqual({ isUsingFacilityEndDate: 'No' });
+    });
+
+    it('should format facilityEndDate to readable format', () => {
+      const facilityEndDate = new Date(2021, 7, 12);
+      const result = facilityFieldsObj({ facilityEndDate: facilityEndDate.toISOString() });
+
+      expect(result).toEqual({ facilityEndDate: '12th August 2021' });
+    });
+
+    it('should format bankReviewDate to readable format', () => {
+      const bankReviewDate = new Date(2021, 7, 12);
+      const result = facilityFieldsObj({ bankReviewDate: bankReviewDate.toISOString() });
+
+      expect(result).toEqual({ bankReviewDate: '12th August 2021' });
```

Comment:
Similar to above, could/should we change this to the following to avoid hard coded date strings?

```js
const expected = format(parseISO(bankReviewDate), LONG_FORM_DATE);

expect(result.bankReviewDate).toEqual(expected);
```

Same in the previous test

Reply:
I think I prefer this test documenting exactly what should be outputted. By doing this if `LONG_FORM_DATE` changes these tests will fail & we'll know that the emails have changed.

Dont feel too strongly though so if you think this should change I'm happy to do so :)

Reply:
I think not having it hard coded would be good and if anything in the function that constructs the date changes, the test will fail without it being updated.

There seem to be a lot of hard coded dates, but most of the dates in the unit tests (searching for `Date).toEqual(`), do _not_ have hard coded dates so I would vote for this 👍